### ISSUE TITLE
Revive lxqt.org (was: lxqt.org is down: Connection reset by peer)

### ISSUE THREAD

**Issue by tptlab** [lxqt/lxqt#1829]
at least the following sites are down:

- https://lxqt.org/
- https://blog.lxqt.org/
- https://weblate.lxqt.org/ (see also #1495)

(following sites are added by @yan12125 as a reminder)

- https://downloads.lxqt.org/
- https://forum.lxqt.org/
- https://git.lxqt.org/ (found via Google. Was it in use?)
- https://lxde.org/
- dev.lxqt.org (the host that was used for managing the weblate instance)

(Some more domain names scanned by https://dnsspy.io/scan)
- imap.lxqt.org
- mail.lxqt.org
- pop.lxqt.org
- smtp.lxqt.org
- wiki.lxqt.org
- www.lxqt.org
```
$ curl -I http://lxqt.org/
curl: (56) Recv failure: Connection reset by peer
$ curl -I https://lxqt.org/
curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to lxqt.org:443 
```
**EDIT:**
 lxqt.org >  https://lxqt-project.org/
weblate.lxqt.org/  > https://translate.lxqt-project.org/

**Comment by yan12125** [comment]
I transferred this issue to this repo as it covers more than the website itself, apparently.

For the forum, I have an idea - create another repo (e.g., "forum") for discussions other than bug reports and feature requests. Issues in the forum repo can be kept open as long as its content is meaningful. If someone creates an issue about bugs or feature requests there, just transfer that issue to an appropriate repo. There is an obvious downside - two repos for issues can be confusing.

**Comment by tsujan** [comment]
> For the forum, I have an idea - create another repo (e.g., "forum") for discussions other than bug reports and feature requests.

Seems very reasonable to me, if it's possible with GitHub.

> There is an obvious downside - two repos for issues can be confusing.

It can be simple: what's in the "forum" won't be considered as a report. If someone wants to report something, he/she should open an issue.

But is GitHub really compatible with such a functionality? I mean GitHub seems to allow users only to open "issues".

**Comment by yan12125** [comment]
If we pretend issues to be forum threads and tell users so, then yes...

**Comment by tsujan** [comment]
> If we pretend issues to be forum threads and tell users so, then yes...

I don't know... It doesn't seem natural. Or, perhaps, it'll seem natural after it's started and used for a while...

Maybe, https://github.com/lxqt/lxqt/issues would be a good place for a forum if it wasn't used for actual bug reports.

**Comment by ito32bit** [comment]
I have another idea.
"Discussions" - but not "Issues".
Is there "Discussions" like https://github.com/vercel/next.js/discussions in  https://github.com/lxqt/lxqt/ ?

**Comment by yan12125** [comment]
Thanks for the info! That looks comparable with the previous self-hosted forum. Looks like the feature is still limited to selected repos 2 months after its announcement [1], as I don't find it in repo settings. Hepo it will come soon.

[1] https://github.blog/2020-05-06-new-from-satellite-2020-github-codespaces-github-discussions-securing-code-in-private-repositories-and-more/

**Comment by stefonarch** [comment]
That looks quite useful, statement from May 6th:

> We are in beta with a few open source communities and will be making Discussions available to other projects soon.

**Comment by tsujan** [comment]
@ito32bit, thanks a lot! "Discussions" is exactly what we need :) When it starts to work, it'll be better than the old forum.

**Comment by jubalh** [comment]
That's a cool feature indeed!

**Comment by ito32bit** [comment]
I hope that Discussions will be enabled in LXQt project.

**Comment by yan12125** [comment]
As a record, I sent a mail to GitHub to ask when that feature will be publicly available.

**Comment by luis-pereira** [comment]
> As a record, I sent a mail to GitHub to ask when that feature will be publicly available.

Great :)

**Comment by yan12125** [comment]
Wow, didn't notice GitHub already replied me in less than 2 hours after I sent the mail!

> Thanks for contacting GitHub Support!
>
> By having a look at out Public Roadmap you can see that GitHub Discussions in planned for Q4 this year:
>
> https://github.com/github/roadmap/projects/1#card-42486645
>
> You can browse our [public roadmap](https://github.com/github/roadmap) to see what we're planning next. We also recommend that you keep an eye on the GitHub Blog for the latest announcements about new features.

**Comment by yan12125** [comment]
An idea about applying a new domain name is discussed https://github.com/lxqt/lxqt/issues/1495#issuecomment-678603563.

> Other point is if we would want a similar setup as lxqt.org, with website and subdomains or only a domain for translation.
I looked some free domain names, lqxt-desktop.foo, lxqt-project.foo, lxqt-translate.foo are available.

I feel lxqt-project.org is better. Some projects (e.g., libqtxdg) can be used for non-desktop applications, and I hope the domain name can be used in places other than the translation system. Any other opinions?

By the way, "free" here means "available for registration", not "free of charge", right? @stefonarch

**Comment by stefonarch** [comment]
> By the way, "free" here means "available for registration", not "free of charge", right?

Yes, the costs would be roughly €5/monthly server and ~14€ for an  *.org domain name.

**Comment by stefonarch** [comment]
So my idea would be:
*  register the domain name (other ideas than **lxqt-project.org**?) with the same company where the server is running
* share the client ID and passwd for hosting and domain renewal at least with other 2 members to prevent a similar issues in future
*  also share ssh access with at least 2 members and of course weblate admin rights too.

I'd be willing to pay at least the first year.

**Comment by yan12125** [comment]
> with the same company where the server is running

Looks like the current Weblate instance (135.181.85.100) runs on a Hetzner VPS? I have no experience with products from that company, but many servers under archlinux.org run on Hetzner, so that sounds a good choice.

> share the client ID and passwd for hosting and domain renewal at least with other 2 members to prevent a similar issue in future
> also share ssh access with at least 2 members and of course weblate admin rights too.

Thanks for combining the list. I think that info should be somewhere easy to find (a file in this repo?) and the current credential holders should be listed on it.

> I'd be willing to pay at least the first year.

Thank you very much. That brings another question - how to handle donations for LXQt? I didn't find an issue discussing about that. Some organizations uses quite a few platforms (e.g., https://keepassxc.org/donate/). I'm not sure we want that many, but picking one from them might be a good start.

**Comment by stefonarch** [comment]
Yeah it's Hetzner, I use them for 2 years for another project and cost/service/helpdesk is eccellent IMO.

> how to handle donations for LXQt?

I'm not 100% sure about it, but from a legal point of view I think to collect donations there has to be at minimum an association, or a company or similar - a legal personality, if donations go to an indivudual person there are taxable as income. Looking atr xfce:

> The Xubuntu project is a team of contributors and not a formal organization. As a result, the project is unable to directly accept monetary donations.



US: https://legalbeagle.com/12191791-legally-donations.html 
EU (Germany): for example LXQt-project e.V would be an registered non-profit association, minimum 7 founder, with a statute, president and some organs, which  could handle an bank account ecc ecc. For opening fisical presence of the leading 3 people or so is required.

Not sure about keepassx but it looks like a indivual responsablity, paypal goes to his name. 
Problems with taxes could arise if someone just decides to donate 10k....

**Comment by stefonarch** [comment]
> I think that info should be somewhere easy to find (a file in this repo?) and the current credential holders should be listed on it.

https://github.com/lxqt/lxqt/wiki/Translation would be a good place  I think

**Comment by stefonarch** [comment]
I'd like to finally  set up the domain and letsencrypt certs.
Any other ideas or lxqt-project.org is fine?
An alternative could be translate-lxqt.cloud or similar with *.cloud which is  only 1.20 € the first year.

**Comment by yan12125** [comment]
Thank you very much for details about donations. The complexity goes beyond my expectation, so I created a dedicated issue for that: https://github.com/lxqt/lxqt/issues/1846

> https://github.com/lxqt/lxqt/wiki/Translation would be a good place I think

That will cover more than translation stuffs, so I created a dedicated page https://github.com/lxqt/lxqt/wiki/Infrastructure-Fellows

> An alternative could be translate-lxqt.cloud or similar using *.cloud which is only 1.20 € the first year.

Domain names that are cheap for the first year are usually expensive since the second year :D

By the way the .cloud TLD sounds a little strange - LXQt are mostly desktop applications rather than cloud ones ;)

**Comment by stefonarch** [comment]
I've running a .cloud TLD, and what you can say most servers are in the cloud ;)
It's just a special offer for 0.99 the first year, then it's still somewhat cheaper than .org

The point is: lxqt-project.org (or similar) would rather be a "full" LXQt website, with translate.lxqt-project.org for the weblate instance, and maybe else too like download.* or blog.* . But as for the forum there is hope for this ["discussion" feature](https://github.com/lxqt/lxqt/issues/1829#issuecomment-664771743) on github. 

So just a domain name for only weblate translations or a "expandable" setup with subdomains  like before?

**Comment by yan12125** [comment]
> I've running a .cloud TLD, and what you can say most servers are in the cloud ;)

You got it ;)

> So just a domain name for only weblate translations or a "expandable" setup with subdomains like before?

I prefer the old expandable setup. Using common domain name (e.g., lxqt-project.org) saves money and maintenance cost - we don't need to register and manage another domain name if we need it one day. Just changing DNS settings is enough.

I'm not sure about Hetzner, but I've used Cloudflare to manage my subdomains. It's free :)

**Comment by stefonarch** [comment]
I don't have a big experience with webservers, domains and subdomains, but I looked in https://www.hetzner.com/dns-console and there  I could add subdomains for free too.
Actually I don't like much (US) gigants like cloudfare- too much power in few hands.
No idea about configuring nginx on the the same machine to serve both the main page and one or more subdomains.

So it lacks just  a consensus about the common domain name.

**Comment by stefonarch** [comment]
Made some tests with another server with subdomains, nginx and https - looks fine doing all with Hetzner.

If I put it this way: Any objections _against_ using **translate.lxqt-project.org**?

**Comment by n4nn31355** [comment]
Have you considered slack or gitter.im instead of the forum? Last one have not so obvious UI\UX, but have rooms\channels, git integration and auth via github.

**Comment by yan12125** [comment]
> Made some tests with another server with subdomains, nginx and https - looks fine doing all with Hetzner.

Thanks a lot! Sorry I don't have much time recently.

> If I put it this way: Any objections against using translate.lxqt-project.org?

@tsujan @luis-pereira any ideas?

Regarding DNS - it is great that Hetzner already supports that! Less accounts to maintain is better 👍 

As a record, I choose Cloudflare for my own server because at that time (early 2018) certbot appears the only mature ACME client and Cloudflare is one of few DNS providers supported by certbot. Now there is a certbot plugin for Hetzner [1] 🎉 

[1] https://github.com/ctrlaltcoop/certbot-dns-hetzner

---

> Have you considered slack or gitter.im instead of the forum? Last one have not so obvious UI\UX, but have rooms\channels, git integration and auth via github.

If I remember it correctly Slack and Gitter are for instant messaging instead of a forum? Currently LXQt uses IRC for instant messaging. I think another instant messaging platform is good if it can sync with IRC.

**Comment by tsujan** [comment]
@yan12125, I'll agree with any decision that may be made here. My knowledge of Weblate is almost zero.

As for the forum, I can't find a Discussions section on https://github.com/lxqt, although GitHub has this page about it: https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-team-discussions

Their description is a little unclear. They say that "public discussions" are possible but only talk about "organization members". Will search the Internet when I find the time.

**Comment by tsujan** [comment]
OK, I found this: https://github.com/orgs/lxqt/teams/lxqt/discussions. But wasn't it always there?! I doubt that non-members can do anything with it.

**Comment by n4nn31355** [comment]
> If I remember it correctly Slack and Gitter are for instant messaging instead of a forum? Currently LXQt uses IRC for instant messaging. I think another instant messaging platform is good if it can sync with IRC.

Yep, but it has threads that can work in most cases. Honestly, IRC isn't user friendly and convenient enough and I believe that it doesn't contribute to a good new developer experience.

**Comment by Chiitoo** [comment]
Public GitHub discussion area is still in the works as far as I understand.

I think this is the roadmap item, but there was something else I read in the past, too... I think: https://github.com/github/roadmap/issues/104

**Comment by yan12125** [comment]
> I'll agree with any decision that may be made here.

@tsujan Well, I mean the new domain name lxqt-project.org ;)

And thanks @Chiitoo for the link. Yes that is a new GitHub feature available to selected repositories only.

Regarding Slack/Gitter: I know Slack has threads. Still far from a forum to me. A possibly good complement for IRC, of course.

**Comment by yan12125** [comment]
By the way, recently I learned that Discourse has a "fork" feature that allows splitting some comments in a thread out and move them to a new thread. That will be a good feature for threads involving multiple related topics like this one ;)

GitHub issues has a similar "Reference in a new issue" feature, but only one comment in the new thread.

**Comment by Chiitoo** [comment]
Yer welcome!

Speaking of forums, my vote would probably go to phpBB, but I guess I'm a bit old fashioned, and I'm sure it's not too popular of a choice these days.

(I have experience in both, using and administering said softare, and I still like it.)

**Comment by stefonarch** [comment]
I could also look in to set a discourse instance (or phpBB which I'm familiar with) up on the server.
In any way: on any forum, discourse or phpBB  or github discussions on has to sign up to partecipate.
I agree it's definitively needed, for keeping basic stuff, comunity and maybe better quite separated from the issues and feature request. Maybe open a dedicated thread about forum, where and how?

BTW I'm about to buy the domain lxqt-project.org next days.

**Comment by stefonarch** [comment]
Done:
https://lxqt-project.org/
https://translate.lxqt-project.org/

**Comment by stefonarch** [comment]
Today I noticed this button in my GH account:

![schermata-08-21-40-48](https://user-images.githubusercontent.com/10681413/101539508-edabb000-399e-11eb-8c00-6e36a2cfd344.png)

Didn't understand much of it atm, how to set it up as forum, it can be enabled  in the settings. 

https://stefonarch.github.io/lxqt-themes/

**Comment by tsujan** [comment]
Yes, I see it too :)

@stefonarch, @yan12125, @luis-pereira
It seems that LXQt is just one click away from having a forum inside GitHub. Would you enable it?

**Comment by yan12125** [comment]
Go ahead and enable it!

tsujan <notifications@github.com> 於 2020年12月9日 週三 05:20 寫道：

**Comment by stefonarch** [comment]
Created an separate issue,  I think there will be some  stuff to discute and decide.
IMHO this issue is here can be closed as solved by lxqt-project.org (nearly).

**Comment by luis-pereira** [comment]
Go ahead.

**Comment by tsujan** [comment]
> Go ahead.

It's enabled and seems like a good forum. Users need to get familiar with it and not to open issues that belong to "Discussions".

**Comment by stefonarch** [comment]
Maybe we could insert in the issue template something like 

> Only for code issues. 
> For other questions, discussion and else please use https://github.com/lxqt/lxqt/discussions

and a link as before on the website, "Forum" doesn't feel right.

**Comment by yan12125** [comment]
> Maybe we could insert in the issue template something like

Should feature requests via issues or discussions? In some other projects those are via issues.

> and a link as before on the website, but "Forum" doesn't feel right, maybe just "Discussions"?

Sounds good!

By the way, I've added https://github.com/lxqt/lxqt/discussions to the topic string of #lxqt IRC channel.

**Comment by stefonarch** [comment]
> Should feature requests via issues or discussions?

https://github.com/lxqt/lxqt/discussions/1881

**Comment by tsujan** [comment]
>  Maybe we could insert in the issue template something like...

There will always be users who don't read and don't care. They need to be told directly and forcefully.

> Should feature requests via issues or discussions?

Issues because codes can be written for them. Let's see it this way: a report that can be closed by a PR belongs to issues. It can be either a feature request or a bug report.

If the feature in question isn't clear enough, it could be discussed in "Discuttions"

**Comment by yan12125** [comment]
> There will always be users who don't read and don't care.

There are also users that are used to issues and don't know about Discussions. Adding one statement brings some benefit and no harm IMO.

**Comment by wxl** [comment]
> By the way, I've added https://github.com/lxqt/lxqt/discussions to the topic string of #lxqt IRC channel.

Can we do that on #lxqt-dev too?

**Comment by yan12125** [comment]
> Can we do that on #lxqt-dev too?

Good idea! Done.

**Comment by stefonarch** [comment]
Just stumbled about [this here](https://news.siduction.org/2021/06/infrastructural-changes/) which sounds familiar:

> Our biggest problem with the situation was having two servers ready, but our domain being beyond our control at this point.

**Comment by tsujan** [comment]
> I wonder how the Siduction team is coping...

Thanks to the practical approach of LXQt members, LXQt quickly recovered from that shock.

**Comment by stefonarch** [comment]
I remembered that in 2023 it could be available, but:

```
Domain Name: lxqt.org
Registry Domain ID: 78dd2ec55a2f431f97def221e104408e-LROR
Registrar WHOIS Server: http://whois.gandi.net
Registrar URL: http://www.gandi.net
**Updated Date: 2023-06-02T23:08:29Z**
Creation Date: 2013-11-25T13:36:06Z
Registry Expiry Date: 2025-11-25T13:36:06Z

```

so it looks it gets paid. There was a legal way I read about to get hands on it, maybe a trial?

**Comment by stefonarch** [comment]
Registrar Information
    Name: Gandi SAS
    Abuse contact email: abuse@gandi.net 
    
    maybe just write them?

**Comment by yan12125** [comment]
> Maybe just write them?

I doubt that works. I may not try it.

Anyway, we should reopen this issue and change the title or open a new issue if we want to regain lxqt.org.

**Comment by stefonarch** [comment]
By chance I stumbled upon this message in the mailing list:
https://sourceforge.net/p/lxde/mailman/lxde-list/?viewmonth=202507&viewday=15

I contacted him, and I think we should take the domain, just to make a redirect to lxqt-project.org

**Comment by yan12125** [comment]
> I think we should take the domain, just to make a redirect to lxqt-project.org

+1 on this

**Comment by medicalwei** [comment]
Hi, holder (or who has permission on lxqt.org domain name) here. Since this issue is closed, I think it is better to either reopen this issue or use a new issue for lxqt.org handover.

Also, now {,www}.lxqt.org is using Gandi's built-in forwarder to lxqt-project.org, it should take effect when TTL is over.

**Comment by yan12125** [comment]
Here you go! Recent discussions in this issue are all about the domain and its owner, so continuing here is fine IMO.

> Also, now {,www}.lxqt.org is using Gandi's built-in forwarder to lxqt-project.org, it should take effect when TTL is over.

Thank you!

I remember lxqt-project.org is using Hetzner. Using the same registrar for lxqt.org simplifies future maintenance. I'm not sure if Hetzner has a similar feature. Using a web server to redirect is still an option.

Possible next actions include:
* Transfer lxqt.org from Gandi to Hetzner
* Get money for paying lxqt.org from https://opencollective.com/lxqt and update the budget table there

**Comment by stefonarch** [comment]
https://lxqt.org works fine, thanks!

This should be relevant:
https://docs.hetzner.com/konsoleh/general/orders/change-of-provider/

**Comment by stefonarch** [comment]
<img width="953" height="83" alt="Image" src="https://github.com/user-attachments/assets/8ad9a045-4f82-402b-a5e7-b70fbc0a21ad" />

<img width="979" height="569" alt="Image" src="https://github.com/user-attachments/assets/b075eb28-1fbf-49b7-ab58-84f5d50a432c" />

Guess that's the right place. Do we need custom DNS server to make a redirect @yan12125 ?

**EDIT** It looks we need this: https://docs.hetzner.com/konsoleh/general/orders/addondomains but I don't really find the Addon domain they're talking about.

**Comment by yan12125** [comment]
> Do we need custom DNS server to make a redirect 

I assume custom DNS servers are for delegating to another platform (ex: Cloudflare) to manage domain names via DNS NS records. If that's the case, we don't need that - managing everything on Hetzner is easier.

> It looks we need this: https://docs.hetzner.com/konsoleh/general/orders/addondomains but I don't really find the Addon domain they're talking about.

It looks like "Addon domain" the "Select Domain" form mentioned above provide the same feature, as they provide the same options.

Most likely @medicalwei can get the auth code from Gandi. Just to remind: that code allows everyone to take over the domain name, so **don't** paste it here.

**Comment by medicalwei** [comment]
I sent the transfer code to @stefonarch through email.

**Comment by stefonarch** [comment]
> I sent the transfer code to [@stefonarch](https://github.com/stefonarch) through email.

Received and domain is ordered, hopefully all fine, thanks!

<img width="659" height="154" alt="Image" src="https://github.com/user-attachments/assets/e044eb2e-a3dd-487f-b0e8-4251db705b80" />

When it's transferred we need to look at https://docs.hetzner.com/konsoleh/account-management/development/redirection-of-a-domain/#redirect-addon-domains-to-the-main-domain

**Comment by yan12125** [comment]
> When it's transferred we need to look at https://docs.hetzner.com/konsoleh/account-management/development/redirection-of-a-domain/#redirect-addon-domains-to-the-main-domain

Thanks, apparently Hetzner does not provide a handy feature like Gandi and requires a web server for such a redirection. There is already an nginx web server running on translate.lxqt-project.org, which may be reused for the redirection.

My two cents on redirection rules:
* Redirect https://weblate.lxqt.org/ to https://translate.lxqt-project.org/
* Redirect all other domains mentioned in https://github.com/lxqt/lxqt/issues/1829#issue-666391920 to https://lxqt-project.org/

**Comment by stefonarch** [comment]
In the meantime I noticed that too, looks it has to be done in `nginx,conf`. Not sure what happens but https://translate.lxqt-project.org/ is really slow to load and navigate atm.

**Comment by stefonarch** [comment]
I read  it can take upon 6 days

<img width="331" height="163" alt="Image" src="https://github.com/user-attachments/assets/a711ef89-c0c7-49b0-b8ad-9667dbd52c99" />

**Comment by yan12125** [comment]
> In the meantime I noticed that too, looks it has to be done in `nginx,conf`. Not sure what happens but https://translate.lxqt-project.org/ is really slow to load and navigate atm.

Yeah I noticed that too. Given some bad seems happening as I mentioned in https://github.com/lxqt/lxqt/issues/2754, using that server for a redirect may not be reliable.

**Comment by stefonarch** [comment]
http://lxqt.org/ is transferred, got the email, @yan12125 do you make the redirect or shall I? Afaik it's enough only redirect those 3 as all other won't have relevant links pointing to them anymore.
https://lxqt.org/
https://blog.lxqt.org/ → https://lxqt-project.org/blog/
https://weblate.lxqt.org/ → https://translate.lxqt-project.org/

**Comment by yan12125** [comment]
> do you make the redirect or shall I?

Nope, I didn't make them yet. Could you help on it? Thanks. Those three redirect are good enough. More can be added in the future if needed.

**Comment by stefonarch** [comment]
I made some short  trials, without success atm.

**Comment by stefonarch** [comment]
Wondering if changing here the record is necessary too, making it point to 135.181.85.100.

<img width="1105" height="446" alt="Image" src="https://github.com/user-attachments/assets/8c47b004-7afc-4c70-9f81-44a433a0afd9" />

**Comment by palinek** [comment]
> Wondering if changing here the record is necessary too, making it point to 135.181.85.100.

If that's the IP where lxqt-project.org is handled, then yes. But you also need to check the configuration of web server handling the traffic - normally there are rules to handle traffic separately per domain. We just need to handle both lxqt.org and lxqt-project.org the same way.

**Comment by palinek** [comment]
But what is that IP? Based on current DNS, the lxqt-project.org is resolved to different IP(s):
```
$ dig @8.8.8.8 lxqt-project.org
 
; <<>> DiG 9.20.11-1-Debian <<>> @8.8.8.8 lxqt-project.org
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 20380
;; flags: qr rd ra; QUERY: 1, ANSWER: 4, AUTHORITY: 0, ADDITIONAL: 1
 
;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;lxqt-project.org.              IN      A
 
;; ANSWER SECTION:
lxqt-project.org.       86400   IN      A       185.199.109.153
lxqt-project.org.       86400   IN      A       185.199.110.153
lxqt-project.org.       86400   IN      A       185.199.111.153
lxqt-project.org.       86400   IN      A       185.199.108.153
 
;; Query time: 44 msec
;; SERVER: 8.8.8.8#53(8.8.8.8) (UDP)
;; WHEN: Tue Jul 29 08:13:32 CEST 2025
;; MSG SIZE  rcvd: 109
```

**Comment by stefonarch** [comment]
Atm I don't remember how the main domain points to lxqt.github.io (which we used again when lxqt.org was down).

**Comment by stefonarch** [comment]
<img width="1107" height="401" alt="Image" src="https://github.com/user-attachments/assets/fa2b34f9-d9a5-47ac-a2e1-1ca07b381523" />

Those are github IPs.

**Comment by palinek** [comment]
Yes, so configure the same way lxqt.org also. Then we also need to update the configuration on github pages.

**Comment by yan12125** [comment]
Using github pages for lxqt.org may not work well. Here are some issues:

* Only one custom domain can be configured for one github repo per https://github.com/orgs/community/discussions/41822. As lxqt-project.org is already configured at https://github.com/lxqt/lxqt.github.io/settings/pages, another github repo is needed to allow using github pages for lxqt.org. If more domains are needed, there will be more github repos, which do not scale well.
* Github pages does not support HTTP 301 refirection yet (https://github.com/orgs/community/discussions/22407). An alternative is using HTML meta tag, which is discouraged by many as it's slow and not good for SEO (https://www.google.com/search?q=meta+refresh+vs+301+redirect&client=firefox-b-m).

My original thought was adding some lines to the nginx config on translate.lxqt-project.org to make it also handle lxqt.org. If there is a better approach, feel free to propose it!

**Comment by stefonarch** [comment]
I think is needs both,  DNS entry _and_ nginx directives, I'll retry now for https://weblate.lxqt.org

**Comment by stefonarch** [comment]
No idea what happened really, the above link points to an outdated website (which seems to be the old clone on the server somehow) while  https://translate.lxqt-project.org/ is offline. I deleted the DNS record and reversed the nginx config but still it's like that.

EDIT: I added the DNS record this morning and I should have set TTL lower than the default (7200) apparently.

**Comment by stefonarch** [comment]
Ok, without taking down anything this time are working now:
http://lxqt.org
http://www.lxqt.org
https://lxqt.org
https://www.lxqt.org

Steps needed:
* Server blocks added on top of `nginx.conf`
* DNS records for lxqt.org at Hetzner changed to point to the server with nginx
* Delete IPV6 AAAA records to make certbot work (lxqt-project.org has no IPv6, do we need that?).
It looks like lxqt.org needed its own certificate.

It looks like do redirect subdomains DNS records for them should be added, but after what happened earlier today I won't try that again atm.

**Comment by yan12125** [comment]
Thank you very much for the efforts! They look mostly good, just with a minor issue:
```
$ curl https://www.lxqt.org/
curl: (60) SSL: no alternative certificate subject name matches target hostname 'www.lxqt.org'
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not establish a secure connection to it. To learn more about this situation and how to fix it, please visit the webpage mentioned above.
```
It's not a big deal, as the same URL works in browsers (tested Firefox). I can have a look next week.

**Comment by tsujan** [comment]
> … as the same URL works in browsers (tested Firefox)

I tested with Chromium; it showed the problem too:

<img width="1356" height="819" alt="Image" src="https://github.com/user-attachments/assets/3848a6bd-4cc6-4554-8f83-630dd785e62d" />

**Comment by stefonarch** [comment]
I think we have anyway an issue with www:
http://www.lxqt.project.org/
https://www.lxqt.project.org/

**Comment by yan12125** [comment]
> I think we have anyway an issue with www: http://www.lxqt.project.org/ https://www.lxqt.project.org/

I assume you meant `www.lxqt-project.org`? That's probably fine, as the www domain for lxqt-project.org never existed. On the other hand, `www.lxqt.org` appears in package metadata on Debian and several Linux distributions ([google search results](https://www.google.com/search?q=%22www.lxqt.org%22&client=firefox-b-m)), so it would be nice to have it fully working.
